package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "design-system-wordpress-child-theme-template",
+  "version": "1.0.0",
+  "description": "Child Theme Template",
+  "author": "govwordpress@gov.bc.ca",
+  "license": "GPL-2.0-or-later",
+  "keywords": [
+    "WordPress",
+    "Block Theme",
+    "BCGov",
+    "DesignSystem",
+    "Gutenberg"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bcgov/design-system-wordpress-child-theme-template.git"
+  },
+  "bugs": {
+    "url": "https://github.com/bcgov/design-system-wordpress-theme#readme",
+    "email": "govwordpress@gov.bc.ca"
+  },
+  "engines": {
```

Comment:
we should either add a .npmrc file or another way of ensuring that the node version is used https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json